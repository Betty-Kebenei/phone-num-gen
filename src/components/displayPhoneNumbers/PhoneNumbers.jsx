import React from 'react';

const PhoneNumbers = () => (<div />);
export default PhoneNumbers;
