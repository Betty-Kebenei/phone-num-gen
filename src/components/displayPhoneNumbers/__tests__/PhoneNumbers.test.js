import React from 'react';
import renderer from 'react-test-renderer';

import PhoneNumbers from '../PhoneNumbers';

test('phonenumber component renders correctly', () => {
  const component = renderer.create(<PhoneNumbers />);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
