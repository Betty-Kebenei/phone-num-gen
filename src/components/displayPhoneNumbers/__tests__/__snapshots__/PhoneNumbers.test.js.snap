// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`phonenumber component renders correctly 1`] = `
<div
  className="displays"
>
  <div
    className="summary"
  >
    <div>
      <p>
        Phone Number Generation Summary
      </p>
      <table>
        <tr>
          <th>
            Total
          </th>
          <td>
            0
          </td>
        </tr>
        <tr>
          <th>
            Maximum Number
          </th>
          <td />
        </tr>
        <tr>
          <th>
            Minimum
          </th>
          <td />
        </tr>
      </table>
    </div>
  </div>
  <div
    className="all-numbers"
  >
    <p>
      There are no phone numbers that have been generated so far
    </p>
  </div>
</div>
`;
